media-video/wireplumber/wireplumber-0.4.10-r1.ebuild

```diff
@@ -93,6 +93,11 @@ src_install() {
 	# will not actually get stored twice until modified.
 	insinto /etc
 	doins -r ${ED}/usr/share/wireplumber
+
+	# Disable both ALSA and bluetooth monitors which implement sound server parts,
+	# media-video/pipewire package should conditionally enable these as needed.
+	sed -i -e "s,alsa_monitor.enabled = true,alsa_monitor.enabled = false,g" "${ED}"/etc/wireplumber/main.lua.d/50-alsa-config.lua || die
```

Comment:
As you might be aware, sed is considered a bad practice, because it is very prone to silently breaking and from what I remember the `|| die` probably doesn't kick in even if the file doesn't exist. Patching should be done by applying patches from FILESDIR.

Reply:
This one is fine, alternative is patch or sed on the source - this can be done if absolutely required.

Reply:
Unless you're editing C files that will be compiled, it's just smoke and mirrors on where you're editing unchanging data/configuration files - that itself is the QA violation no matter how you spin it.

Reply:
OK I now amended that commit to replace `sed` with patch file, hope this fits.

Reply:
Yes, that particular objection is cleared. I still think you should only install a drop-in which sets the enabled state to false, which should work if you choose the right number to assign to the drop in, making all these patches unnecessary in the first place.